subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h

```diff
@@ -0,0 +1,164 @@
+#define HAL_RADIO_ENABLE_TX_ON_TICK_PPI HAL_RADIO_ENABLE_ON_TICK_PPI
+#define HAL_RADIO_ENABLE_RX_ON_TICK_PPI HAL_RADIO_ENABLE_ON_TICK_PPI
+#define HAL_RADIO_RECV_TIMEOUT_CANCEL_PPI 3
+#define HAL_RADIO_DISABLE_ON_HCTO_PPI 4
+#define HAL_RADIO_END_TIME_CAPTURE_PPI 5
+
+#endif /* defined(CONFIG_BT_CTLR_TIFS_HW) || !defined(CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER) */
+
+/* Start event timer on RTC tick wire the RTC0 EVENTS_COMPARE[2] event to
+ * EVENT_TIMER  TASKS_START task.
+ */
+#define HAL_EVENT_TIMER_START_PPI 6
+
+/* Capture event timer on Radio ready, wire the RADIO EVENTS_READY event to the
+ * EVENT_TIMER TASKS_CAPTURE[<radio ready timer>] task.
+ */
+#define HAL_RADIO_READY_TIME_CAPTURE_PPI 7
+
+/* Trigger encryption task upon address reception:
+ * wire the RADIO EVENTS_ADDRESS event to the CCM TASKS_CRYPT task.
+ *
+ * PPI channel 25 is pre-programmed with the following fixed settings:
+ *   EEP: RADIO->EVENTS_ADDRESS
+ *   TEP: CCM->TASKS_CRYPT
+ */
+#define HAL_TRIGGER_CRYPT_PPI 25
+
+/* Trigger automatic address resolution on Bit counter match:
+ * wire the RADIO EVENTS_BCMATCH event to the AAR TASKS_START task.
+ *
+ * PPI channel 23 is pre-programmed with the following fixed settings:
+ *   EEP: RADIO->EVENTS_BCMATCH
+ *   TEP: AAR->TASKS_START
+ */
+#define HAL_TRIGGER_AAR_PPI 23
+
+/* Trigger Radio Rate override upon Rateboost event. */
+#if defined(CONFIG_HAS_HW_NRF_RADIO_BLE_CODED)
+#define HAL_TRIGGER_RATEOVERRIDE_PPI 14
+#endif /* CONFIG_HAS_HW_NRF_RADIO_BLE_CODED */
+
+#define HAL_ENABLE_PALNA_PPI  15
+#define HAL_DISABLE_PALNA_PPI 16
+
+#if defined(HAL_RADIO_FEM_IS_NRF21540)
+#define HAL_ENABLE_FEM_PPI  4
+#define HAL_DISABLE_FEM_PPI 5
+#endif /* HAL_RADIO_FEM_IS_NRF21540 */
+
+#if !defined(CONFIG_BT_CTLR_TIFS_HW)
+/* PPI setup used for SW-based auto-switching during TIFS. */
+
+#if !defined(CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER)
+
+/* Clear SW-switch timer on packet end:
+ * wire the RADIO EVENTS_END event to SW_SWITCH_TIMER TASKS_CLEAR task.
+ *
+ * Note: this PPI is not needed if we use a single TIMER instance in radio.c
+ */
+#define HAL_SW_SWITCH_TIMER_CLEAR_PPI 8
+
+#else /* !CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER */
+
+/* Clear event timer (sw-switch timer) on Radio end:
+ * wire the RADIO EVENTS_END event to the
+ * EVENT_TIMER TASKS_CLEAR task.
+ *
+ * Note: in nRF52X this PPI channel is forked for both capturing and clearing
+ * timer on RADIO EVENTS_END.
+ */
+#define HAL_SW_SWITCH_TIMER_CLEAR_PPI HAL_RADIO_END_TIME_CAPTURE_PPI
+
+#endif /* !CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER */
+
+/* Wire a SW SWITCH TIMER EVENTS_COMPARE[<cc_offset>] event
+ * to a PPI GROUP TASK DISABLE task (PPI group with index <index>).
+ * 2 adjacent PPIs (9 & 10) and 2 adjacent PPI groups are used for this wiring;
+ * <index> must be 0 or 1. <offset> must be a valid TIMER CC register offset.
+ */
+#if defined(CONFIG_SOC_NRF52805)
+#define HAL_SW_SWITCH_GROUP_TASK_DISABLE_PPI_BASE 2
+#else /* -> !defined(CONFIG_SOC_NRF52805) */
```

Comment:
nit-pick, we usually write #else this way in controller:
```suggestion
#else /* !CONFIG_SOC_NRF52805 */
```

Reply:
done